frontend/express/public/core/geo-countries/javascripts/countly.views.js

```diff
@@ -0,0 +1,291 @@
+    computed: {
+        data: function() {
+            return this.$store.state.countlyCities.data;
+        },
+        totalData: function() {
+            return this.$store.state.countlyCountry.data;
+        },
+        chooseProperties: function() {
+            var totals = this.countryTotals;
+
+            return [
+                {"value": "t", "label": CV.i18n('common.table.total-sessions'), "trend": totals.t.trend, "number": countlyCommon.getShortNumber(totals.t.total || 0), "trendValue": totals.t.change},
+                {"value": "u", "label": CV.i18n('common.table.total-users'), "trend": totals.u.trend, "number": countlyCommon.getShortNumber(totals.u.total || 0), "trendValue": totals.u.change},
+                {"value": "n", "label": CV.i18n('common.table.new-users'), "trend": totals.n.trend, "number": countlyCommon.getShortNumber(totals.n.total || 0), "trendValue": totals.n.change}
+            ];
+        },
+        countryTotals: function() {
+            var totals = this.data.totals;
+            totals = totals || {"u": {}, "t": {}, "n": {}};
+            totals.t = totals.t || {};
+            totals.u = totals.u || {};
+            totals.n = totals.n || {};
+            return totals;
+        },
+        countriesData: function() {
+            var data = {};
+            var totals = this.countryTotals;
+            var value = totals.t.total || 1;
+            data[this.$route.params.region] = {"value": value};
+            return data;
+        },
+        citiesData: function() {
+            var data = this.data.cities || [];
+            var selectedProperty = this.$store.state.countlyCities.selectedProperty || "t";
+            var cityData = {};
+            if (data.length > 0) {
+                for (var k = 0; k < data.length; k++) {
+                    cityData[data[k].city] = {"value": data[k][selectedProperty]};
+                }
+            }
+            var retObj = {};
+            retObj[this.$route.params.region] = cityData;
+            return retObj;
+        },
+        cityTable: function() {
+            return this.data.table || [];
+        },
+        selectedPropertyTitle: function() {
+            var selectedProperty = this.$store.state.countlyCities.selectedProperty || "t";
+            if (selectedProperty === 't') {
+                return CV.i18n('common.table.total-sessions');
+            }
+            else if (selectedProperty === 'u') {
+                return CV.i18n('common.table.total-users');
+            }
+            else if (selectedProperty === 'n') {
+                return CV.i18n('common.table.new-users');
+            }
+        },
+        selectedProperty: {
+            set: function(value) {
+                this.$store.dispatch('countlyCities/onSetSelectedProperty', value);
+            },
+            get: function() {
+                return this.$store.state.countlyCities.selectedProperty;
+            },
+            dropdownsDisabled: function() {
```

Comment:
Probably a mistake. I think `dropdownsDisabled` is supposed to be a computed field while you define it in the selectedBrowser field.

I noticed this one of Pınar's PRs too. Is this coming from somewhere?